### DISCUSSION TITLE
Material Weight output

### DISCUSSION THREAD

**Discussion by ninarular** [cwapi3d/cwapi3dpython#185] [Q&A]
Im currently working on a plugin that allows our team to create profiles on a CLT Material class for a group of various CLT Materials. I was wondering what the output for the Material controller get material weight function was, as it doesn't match up with the lb/ft^3 value found in our materials list. Is it referencing another unit and if so is there a way to import the density as an lb/ft^3 value

**Comment by ninarular** [comment]
![image](https://github.com/cwapi3d/cwapi3dpython/assets/110792376/d13b4aff-32b5-46a1-a471-d844003eba42)
![image](https://github.com/cwapi3d/cwapi3dpython/assets/110792376/34be8a5d-6dfa-4f5e-91a7-f33bf6ec371e)

**Comment by jspaquet** [comment]
All units in CwAPI3D are always metric. So in this case it would be weight/volume, so most likely kN/m3 but double check the output to make sure (it might be kg/m3 but I doubt it since the other weight outputs are in kN).
